csi/src/meta_data.rs

```diff
@@ -775,7 +846,7 @@ impl MetaData {
                         RpcStatusCode::INVALID_ARGUMENT,
                         anyhow!(format!(
                             "source snapshot ID={} and name={} has size={} \
-                                greater than requested volume size={}",
+                            greater than requested volume size={}",
```

Comment:
extra 4 spaces indentation before the second text line

Reply:
Done